share/completions/brew.fish

```diff
@@ -1,3 +1,8 @@
+function __fish_brew_is_subcommand_services
+  __fish_brew_using_command services
+  __fish_brew_using_command $argv[1] 3
```

Comment:
The first call here will be completey without effect.

You'll want to have an `and` in the first line - `and __fish_brew_using_command $argv[1] 3`.